### DISCUSSION TITLE
Running Project Origin on Azure

### DISCUSSION THREAD

**Discussion by Guy-Heylens-Elia** [project-origin/.github#21] [Q&A]
Hello, 

I am trying to setup Project Origin on an Azure K8S environment for development.  
For this I use the HELM charts and this guide (https://artifacthub.io/packages/helm/project-origin/project-origin-registry).  

According the guide we need to create an issuer key and put the PublicKey in the values file. 

After that I run the command to install chart and everything should start up. Unfortunately there is an issue in the Electricity Server, it can not figure out the PublicKey. And I get following error:
![image](https://github.com/project-origin/.github/assets/104833610/8a553eff-f398-4fb0-a261-acc357d11aa1)

I have tried generating the Issuer key on several different machines, but still get the same error.

KInd regards,

Guy

**Comment by AndreAlnor** [comment]
Hi Guy, I noticed this question has been here for a while, with no answer in Gihub. Did you get help to solve the issue?

**Reply by Guy-Heylens-Elia** [reply]
Sure, with some help of Martin it is running.

**Reply by AndreAlnor** [reply]
Awesome! I'll close this topic then. Have a great day :)
